tests/webapp/api/test_push_api.py

```diff
@@ -125,36 +125,6 @@ def test_push_list_single_long_revision(client, eleven_jobs_stored, test_reposit
     )
 
 
-def test_push_list_single_long_revision_stored_long(client, sample_push, test_repository):
```

Comment:
This poorly named test was ACTUALLY testing fetching a push by a sub-commit.  So no longer needed.